frontend/src/components/MapView/AlertForm/__snapshots__/index.test.tsx.snap

```diff
@@ -44,11 +44,594 @@ exports[`renders as expected 1`] = `
           >
             Hazard Layer
           </mock-typography>
-          <mock-layer-dropdown
```

Comment:
I'm surprised that this test changed so much. Maybe some modules are not mocked properly anymore? Could you investigate @HarryMytilinaios ?

Reply:
of course